rust/kernel/types.rs

```diff
@@ -144,6 +144,22 @@ impl<T> PointerWrapper for *mut T {
     }
 }
 
+impl PointerWrapper for () {
+    type Borrowed<'a> = ();
+
+    fn into_pointer(self) -> *const c_types::c_void {
+        0usize as _
```

Comment:
Maybe use 1 instead? There may be places where C code checks against a null pointer. In addition it is less dangerous when accidentally turning this pointer into a reference. By the way this code can also be written as `core::ptr::null()`.

Reply:
Good idea, I'll change this.